anda/desktops/ctwm/ctwm.spec

```diff
@@ -0,0 +1,54 @@
+Summary: Lightweight window manager with virtual workspaces
+Name: ctwm
+Version: 4.1.0
+Release: 1
+URL: https://ctwm.org
+BuildRequires: libx11-devel libjpg-devel libxt-devel libxext-devel libxmu-devel libxpm-devel cmake
+Source0: https://www.ctwm.org/dist/%{name}-%{version}.tar.gz
```

Comment:
So the annoying part is, we probably want to download sources directly from upstream (due to xz incident we should have learnt our lesson)

ctwm sources do not use git, they use bazaar (totally incompatible), commits are called revisions

supposedly we can track the version in update.rhai using this file https://bazaar.launchpad.net/~ctwm/ctwm/trunk/annotate/head:/VERSION
Downside is we need to regex the webpage to get the version
and then we need to grab the revision (710.1.2), save it in the spec using rpm.global(), then finally we have the tarball at https://bazaar.launchpad.net/~ctwm/ctwm/trunk/tarball/710.1.2

I might do this later